app.py

```diff
@@ -1,41 +1,55 @@
 from flask import Flask, redirect, render_template, request, url_for
+import trello_cards
 import session_items as session
+import os
+from card import Card
```

Comment:
I get an error from this line when running locally.
* Did you forget to include a `card.py` in this review?

Reply:
It's push now.